CHANGES.md

```diff
@@ -24,6 +24,7 @@ Language Improvements:
 - enh(typescript/javascript/coffeescript/livescript) derive ECMAscript keywords from a common foudation (#2518) [Josh Goebel][]
 - enh(typescript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Josh Goebel][]
 - enh(javascript) add setInterval, setTimeout, clearInterval, clearTimeout (#2514) [Vania Kucher][]
+- enh(cpp) add pair, make_pair, priority_queue as built-ins (#2538) [Linhk1606][]
```

Comment:
You need to add a reference on line ~36 to tell it where your name should link to.